index.html

```diff
@@ -59,6 +59,12 @@ <h1 data-l10n-id="app_title">Studio</h1>
       <gaia-header>
         <h1></h1>
       </gaia-header>
+      <div class="flex">
```

Comment:
We're having weird double scroll bars issues.
It's because we want the header to stay on top and the rest to scroll.
Before this patch, "the rest" was the gaia-list, and we did [1], but know we have the window and the list that are scrollable while on this panel.

We probably need to add a container for the buttons and the list and update the rule to match it.

[1] https://github.com/fxos/studio/blob/master/css/app.css#L64